From 0d54fe501f0e2745df77fc6f611836102a52cb2b Mon Sep 17 00:00:00 2001
From: Eric Sauvageau <merlin@asuswrt-merlin.net>
Date: Fri, 14 Aug 2026 01:50:02 -0400
Subject: [PATCH] webui: remove empty popup hints for TTL options on MultiWAN
 page

(cherry picked from commit 27fc90dfa521c5079f1d4bb44ca0588df7c09e36)
---
 .../sysdep/FUNCTION/MULTISERVICE_WAN/Advanced_WAN_Content.asp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/release/src/router/www/sysdep/FUNCTION/MULTISERVICE_WAN/Advanced_WAN_Content.asp b/release/src/router/www/sysdep/FUNCTION/MULTISERVICE_WAN/Advanced_WAN_Content.asp
index 634691b67f..66aeacbd8b 100644
--- a/release/src/router/www/sysdep/FUNCTION/MULTISERVICE_WAN/Advanced_WAN_Content.asp
+++ b/release/src/router/www/sysdep/FUNCTION/MULTISERVICE_WAN/Advanced_WAN_Content.asp
@@ -4408,14 +4408,14 @@ function change_wizard(o, id){
 										</td>
 									</tr>
 									<tr>
-										<th><a class="hintstyle" href="javascript:void(0);" onClick=""><#Extend_TTL_Value#></a></th>
+										<th><#Extend_TTL_Value#></th>
 										<td>
 											<input type="radio" name="ttl_inc_enable" class="input" value="1" <% nvram_match("ttl_inc_enable", "1", "checked"); %>><#checkbox_Yes#>
 											<input type="radio" name="ttl_inc_enable" class="input" value="0" <% nvram_match("ttl_inc_enable", "0", "checked"); %>><#checkbox_No#>
 										</td>
 									</tr>	
 									<tr>
-										<th><a class="hintstyle" href="javascript:void(0);" onClick=""><#Spoof_TTL_Value#></a></th>
+										<th><#Spoof_TTL_Value#></th>
 										<td>
 											<input type="radio" name="ttl_spoof_enable" class="input" value="1" <% nvram_match("ttl_spoof_enable", "1", "checked"); %>><#checkbox_Yes#>
 											<input type="radio" name="ttl_spoof_enable" class="input" value="0" <% nvram_match("ttl_spoof_enable", "0", "checked"); %>><#checkbox_No#>
-- 
2.25.1

